protocol/test/forge/v2/strategies/DsrBaseStrategy.t.sol

```diff
@@ -63,20 +70,40 @@ contract DsrBaseStrategyTestAdmin is DsrBaseStrategyTestBase {
         _setUp();
     }
 
+    function test_dsr_interest_equivalence() public {
+        uint256 dsrRate = 1.01e18;
+        uint256 equivalentDusdRate = ud(dsrRate).ln().unwrap();
+        assertEq(equivalentDusdRate, 0.009950330853168072e18);
+
+        dsrRate = 1.0349e18;
+        equivalentDusdRate = ud(dsrRate).ln().unwrap();
+        assertEq(equivalentDusdRate, 0.034304803691990293e18);
+    }
+
     function test_initalization() public {
-        assertEq(strategy.executors(executor), true);
-        assertEq(strategy.rescuers(rescuer), true);
+        assertEq(strategy.executor(), executor);
+        assertEq(strategy.rescuer(), rescuer);
         assertEq(strategy.apiVersion(), "1.0.0");
         assertEq(strategy.strategyName(), "DsrBaseStrategy");
         assertEq(strategy.strategyVersion(), "1.0.0");
         assertEq(address(strategy.treasuryReservesVault()), address(trv));
-        assertEq(address(strategy.stableToken()), address(dai));
-        assertEq(address(strategy.internalDebtToken()), address(dUSD));
+        assertEq(address(strategy.daiToken()), address(dai));
         ITempleStrategy.AssetBalanceDelta[] memory adjs = strategy.manualAdjustments();
         assertEq(adjs.length, 0);
 
         assertEq(address(strategy.daiJoin()), address(daiJoin));
         assertEq(address(strategy.pot()), address(pot));
+        assertEq(strategy.RAY(), 1e27);
+    }
+
+    function test_setTreasuryReservesVault() public {
+        vm.startPrank(executor);
+        TreasuryReservesVault trv2 = new TreasuryReservesVault(rescuer, executor, address(tpiOracle));
+        strategy.setTreasuryReservesVault(address(trv2));
+        assertEq(address(strategy.treasuryReservesVault()), address(trv2));
+
```

Comment:
nit: although checked in `AbstractStrategy`, I guess it's also ok to check emit here?

Reply:
Fixed in https://github.com/TempleDAO/temple/pull/814/commits/3933dcacf45c04e2bb680b56d4921a0d6b6ba65a